src/services/api/url-resolver/url.resolver.service.ts

```
@@ -0,0 +1,865 @@
+        return result;
+      }
+      case URL_PATHS.USER:
+        if (pathElements.length < 2) {
+          throw new ValidationException(
+            `Invalid URL: ${url}`,
+            LogContext.URL_RESOLVER
+          );
+        }
+        const user = await this.userLookupService.getUserByNameIdOrFail(
+          pathElements[1]
+        );
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)